packages/shared/components/popups/SendConfirmationPopup.svelte

```diff
@@ -175,9 +182,16 @@
         {/if}
     </div>
     <popup-buttons class="flex flex-row flex-nowrap w-full space-x-4">
-        <Button classes="w-full" secondary onClick={onBack} disabled={$isTransferring}
-            >{localize('actions.back')}</Button
-        >
+        {#if disableBack}
+            <Button classes="w-full" secondary onClick={onCancel} disabled={$isTransferring}>
+                {localize('actions.cancel')}
+            </Button>
+        {:else}
+            <Button classes="w-full" secondary onClick={onBack} disabled={$isTransferring}>
+                {localize('actions.back')}
+            </Button>
+        {/if}
+
```

Comment:
We can shorten this to:
```svelte
<Button classes="w-full" secondary onClick={disableBack ? onCancel : onBack} disabled={$isTransferring}>
    {localize(`actions.${disableBack ? 'cancel' : 'back'}`)}
</Button>
```

Reply:
I think this is less readable, in my version you can clearly see the two different components to be displayed